greenback-kit-jackson/src/main/java/com/greenback/kit/jackson/JacksonGreenbackCodec.java

```diff
@@ -43,6 +43,7 @@ public JacksonGreenbackCodec() {
             .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
             .setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE)
             .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+            .enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
```

Comment:
When would this be true?  If you found its true somewhere, we should redesign our model since I can't think of when we'd have a single object that should be an array.